pkg/services/ngalert/sender/sender.go

```diff
@@ -114,7 +114,9 @@ func NewExternalAlertmanagerSender(l log.Logger, reg prometheus.Registerer, opts
 		s.logger.Error("failed to register service discovery metrics", "error", err)
 		return nil, err
 	}
-	s.sdManager = discovery.NewManager(sdCtx, s.logger, prometheus.NewRegistry(), sdMetrics)
+	// Convert s.Logger to slog.Logger using the adapter
+	slogLogger := toSlogLogger(s.logger)
+	s.sdManager = discovery.NewManager(sdCtx, slogLogger, prometheus.NewRegistry(), sdMetrics)
```

Comment:
Prometheus has changed `discovery.NewManager` signature in new version: https://github.com/prometheus/prometheus/pull/14906